### ISSUE TITLE
Website issue

### ISSUE THREAD

**Issue by ofluffydev** [biojava/biojava#1102]
The API documentation (for all versions) has a broken search function. It appears to send you to the root domain instead of /docs. Please look into it, this makes navigating the JavaDocs very annoying.

[This](https://stackoverflow.com/questions/52326318/maven-javadoc-search-redirects-to-undefined-url) may be related, but I can't know for sure without knowing the underlying system for Biojava's website.

**Comment by josemduarte** [comment]
Good catch, thanks for reporting. I'll see if I can have a look. And will also push the latest docs.

**Comment by josemduarte** [comment]
Now fixed. And latest docs published at: https://biojava.org/docs/api/
